python/hsfs/feature_group.py

```diff
@@ -1647,7 +1647,7 @@ def save(
                   feature group.
                 * key `wait_for_job` and value `True` or `False` to configure
                   whether or not to the save call should return only
-                  after the Hopsworks Job has finished. By default it waits.
+                  after the Hopsworks Job has finished. By default it does not wait.
```

Comment:
Can we add a Note admonition above to highlight from which version the default changed? or write it here.

Reply:
I am confused I was expecting a new argument in save/insert? right now you are just changing the default? is this what you agreed on with @jimdowling

Reply:
i discussed this solution with him

Reply:
Seems like we indeed wanted a new parameter, so I added a "wait"= parameter to fg.save/.insert() now